### PR TITLE
Fixed technical and editorial issues

### PR DESCRIPTION
**Pull request by drpayyne**
## Purpose of this pull request
This pull request (PR) address the following changes in the affected page
-  Removed "Possible values" phrase, as the column "Values" by itself depicts the possible values
-  Better terminology with example for `class` for `template` attributes
-  Fixed inconsistent casing
-  Fixed inconsistency in "`true` or `false`" column values throughout page
-  Fixed table rendering bug (image attached below)
image

## Affected DevDocs pages
-  https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/frontend-dev-guide/layouts/xml-instructions.md [hunk 1]
```diff
@@ -60,11 +60,11 @@ We recommend always adding a `name` to blocks. Otherwise, it is given a random n
 
 | Attribute | Description | Values | Required? |
 |:------- |:------ |:------ |:------ |
-| `class` | Name of a class that implements rendering of a particular block. An object of this class is responsible for actual rendering of block output. | class name | no |
+| `class` | Name of a class that implements rendering of a particular block. An object of this class is responsible for actual rendering of block output. | Fully qualified class name E.g. `Vendor\Module\Block\Class` | no |
```
Comment (keharper) [inline guides/v2.3/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
| `class` | Name of a class that implements rendering of a particular block. An object of this class is responsible for actual rendering of block output. | A fully-qualified class name, such as `Vendor\Module\Block\Class` | no |
```

File: guides/v2.3/frontend-dev-guide/layouts/xml-instructions.md [hunk 2]
```diff
@@ -60,11 +60,11 @@ We recommend always adding a `name` to blocks. Otherwise, it is given a random n
...
-| `template` | A template that represents the functionality of the block to which this attribute is assigned. | template file name | no |
+| `before` | Used to position the block before an element under the same parent. The element name or alias name is specified in the value. Use dash (-) to position the block before all other elements of its level of nesting. See [before and after attributes](#fedg_xml-instrux_before-after) for details. | Element name or dash (-) | no |
+| `after` | Used to position the block after an element under the same parent. The element name or alias name is specified in the value. Use dash (-) to position the block after all other elements of its level of nesting. See [before and after attributes](#fedg_xml-instrux_before-after) for details. | Element name or dash (-) | no |
+| `template` | A template that represents the functionality of the block to which this attribute is assigned. | Vendor_Module::path/to/template.phtml (Scope is already in `templates` directory of module) | no |
```
Comment (keharper) [inline guides/v2.3/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
| `template` | A template that represents the functionality of the block to which this attribute is assigned. | `Vendor_Module::path/to/template.phtml` (Scope is already in the `templates` directory of the module) | no |
```

File: guides/v2.3/frontend-dev-guide/layouts/xml-instructions.md [hunk 3]
```diff
@@ -165,7 +165,7 @@ Calls public methods on the block API.
 
 | Attribute | Description | Values | Required? |
 |:------- |:------ |:------ |:------ |
-| `method` | Name of the public method of the block class this tag is located in that is called during block generation. | block method name | yes |
+| `method` | Name of the public method of the block class this tag is located in that is called during block generation. | Method name in block | yes |
```
Comment (keharper) [inline guides/v2.3/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
| `method` | The public method that is called during block generation. | The method name in the block | yes |
```

File: guides/v2.3/frontend-dev-guide/layouts/xml-instructions.md [hunk 4]
```diff
@@ -179,8 +179,8 @@ To pass parameters to a block use the [`<argument></argument>`](#argument) instr
 
 | Attribute | Description | Values | Required? |
 |:------- |:------ |:------ |:------ |
-| `remove` | Allows to remove or cancel the removal of the element. When a container is removed, its child elements are removed as well. | true/false | no |
-| `display` | Allows you to disable rendering of specific block or container with all its children (both set directly and by reference). The block's/container's and its children' respective PHP objects are still generated and available for manipulation. | true/false | no |
+| `remove` | Allows to remove or cancel the removal of the element. When a container is removed, its child elements are removed as well. | `true` or `false` | no |
+| `display` | Allows you to disable rendering of specific block or container with all its children (both set directly and by reference). The block's/container's and its children' respective PHP objects are still generated and available for manipulation. | `true` or `false` | no |
```
Comment (keharper) [inline guides/v2.3/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
| `display` | Allows you to disable rendering of specific block or container with all its children (both set directly and by reference). The PHP objects of the block or container and its children are still generated and available for manipulation. | `true` or `false` | no |
```

**Pull request merged**